docs/content/docs/tutorials/memorydb-example.md

```diff
@@ -0,0 +1,288 @@
+---
+title: "Deploy MemoryDB Cluster Using the ACK MemoryDB Controller"
+description: "Create managed MemoryDB Cluster from a Amazon Elastic Kubernetes Service (EKS) deployment."
+lead: "Create and use MemoryDB for Redis using Amazon Elastic Kubernetes Service (EKS)."
+draft: false
+menu:
+  docs:
+    parent: "tutorials"
+weight: 42
+toc: true
+---
+
+The ACK service controller for Amazon MemoryDB for Redis lets you manage MemoryDB Cluster directly from Kubernetes.
+
+This guide will show you how to create and connect to [MemoryDB for Redis](https://aws.amazon.com/memorydb/) through Kubernetes.
+
+## Setup
+
+Although it is not necessary to use Amazon Elastic Kubernetes Service (Amazon EKS) with ACK, this guide assumes that you have access to an Amazon EKS cluster. If this is your first time creating an Amazon EKS cluster, see [Amazon EKS Setup](https://docs.aws.amazon.com/deep-learning-containers/latest/devguide/deep-learning-containers-eks-setup.html). For automated cluster creation using `eksctl`, see [Getting started with Amazon EKS - `eksctl`](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-eksctl.html) and create your cluster with Amazon EC2 Linux managed nodes.
+
+### Prerequisites
+
+This guide assumes that you have:
+
+- Created an EKS cluster with Kubernetes version 1.18 or higher.
+- Setup the [Amazon VPC Container Network Interface (CNI) plugin for Kubernetes](https://docs.aws.amazon.com/eks/latest/userguide/managing-vpc-cni.html) for the EKS Cluster.
+- AWS IAM permissions to create roles and attach policies to roles.
+- Installed the following tools on the client machine used to access your Kubernetes cluster:
+  - [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv1.html) - A command line tool for interacting with AWS services.
+  - [kubectl](https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html) - A command line tool for working with Kubernetes clusters.
+  - [eksctl](https://docs.aws.amazon.com/eks/latest/userguide/eksctl.html) - A command line tool for working with EKS clusters.
+  - [Helm 3.7+](https://helm.sh/docs/intro/install/) - A tool for installing and managing Kubernetes applications.
```

Comment:
I think this would be the best time to start recommending Helm 3.8. It has native support for OCI registries like we are using. We don't need to do the whole `helm pull` + `untar` + `helm install`. We can just directly use `helm install` and pass it the OCI image URI!

Reply:
I think `untar` is currently needed because we use local files to delete the crds in cleanup [step](https://github.com/aws-controllers-k8s/community/blob/main/docs/content/docs/user-docs/cleanup.md#delete-all-crds).

Maybe we can delete crds by name instead of files and then get rid of untar

Reply:
I have kept the Helm3.7 specific instructions for now.
I think it can be updated later across all the examples files.